internal/api/mfa.go

```diff
@@ -132,7 +149,7 @@ func (a *API) enrollPhoneFactor(w http.ResponseWriter, r *http.Request, params *
 		ID:           factor.ID,
 		Type:         models.Phone,
 		FriendlyName: factor.FriendlyName,
-		Phone:        string(factor.Phone),
+		Phone:        params.Phone,
```

Comment:
`factor.Phone` is e.164 (e.g. +6112343242) and the params would be validated but in the same format (e.g. 6112343242)

Guessing folks would probably want to use the original version